config.go

```diff
@@ -34,6 +37,11 @@ type Config struct {
 	ID         peer.ID
 	PrivateKey crypto.PrivKey
 
+	// Swarm key for private network
+	SwarmSecret string
+	// Fingerprint for private network
+	PNetFingerprint []byte
```

Comment:
Right, I had this in my notes to address at some point, and I agree that it doesn't belong here. It was originally a minor convenience for passing it from the `makeHost()` function.